addons/website/views/snippets/snippets.xml

```diff
@@ -539,6 +539,10 @@
             <we-button class="o_we_bg_brand_primary o_grid" data-add-element="text" title="Text" data-name="text">Text</we-button>
             <we-button class="o_we_bg_brand_primary o_grid" data-add-element="button" title="Button" data-name="button">Button</we-button>
         </we-button-group>
+        <we-row string="Spacing (Y, X)">
+            <we-input data-dependencies="grid_mode" data-select-style="" data-css-property="row-gap" data-unit="px" data-prevent-important="true" data-apply-to=".row"/>
+            <we-input data-dependencies="grid_mode" data-select-style="" data-css-property="column-gap" data-unit="px" data-prevent-important="true" data-apply-to=".row"/>
+        </we-row>
```

Comment:
Maybe time to not duplicated that code? You could extract the common layout part into a view no? :)